qemu/tests/cfg/qsd_live_snapshot.cfg

```diff
@@ -0,0 +1,37 @@
+- qsd_live_snapshot:
+    type = qsd_live_snapshot
+    start_vm = no
+    images + = "stg1"
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    base_tag = "stg1"
+    snapshot_tag = sn1
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+    device = "fmt_stg1"
+    format = qcow2
+    rebase_mode = unsafe
+    only Linux
+    !Host_RHEL.m7:
+        node = "fmt_stg1"
+        overlay = "drive_sn1"
+        qemu_force_use_drive_expression = no
+
+    # QSD attributes
+    qsd_namespaces = "qsd1"
+    qsd_images_qsd1 = "stg1"
+    # Image attributes
+    image_name_stg1 = images/stg1
+    image_size_stg1 = 2G
+    image_size_sn1 = ${image_size_stg1}
+    drive_format_stg1 = vhost-user-blk-pci
+    image_vubp_props_stg1=  {"num-queues":1}
+    # Memory defination
+    vm_mem_backend = memory-backend-memfd
+    vm_mem_share = yes
+
+    qsd_image_format_stg1 = {"driver":"qcow2","read-only":false,"detect-zeroes"
+    qsd_image_format_stg1 += :"unmap","discard":"unmap"}
+    qsd_image_export_stg1 = {"type":"vhost-user-blk"}
```

Comment:
suggest to add these two params for sn1.

Why? for the 2nd qsd, in fact, we export the sn1, I see that in your code you use the confs of stg1 + sn1 file to start the 2nd qsd, in this way, we may write less codes, but it's too confused to understand the 'internals', and it's not a regular process to handle the image and its snapshot

Reply:
qsd_image_format
qsd_image_export

Reply:
Updated as your suggestion, thanks.